llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -4228,7 +4237,7 @@ static SDValue lowerBUILD_VECTOR(SDValue Op, SelectionDAG &DAG,
   // Cap the cost at a value linear to the number of elements in the vector.
   // The default lowering is to use the stack.  The vector store + scalar loads
   // is linear in VL.  However, at high lmuls vslide1down and vslidedown end up
-  // being (at least) linear in LMUL.  As a result, using the vslidedown
+  // being (at least) linear in LMUL.  As a resultdedown
```

Comment:
Something bad happened to this comment.

Reply:
Indeed, good catch. Should be addressed.